fars/booking/test_bookables.py

```diff
@@ -0,0 +1,21 @@
+from django.test import TestCase
+from booking.models import Bookable
+
+class BookableTestCase(TestCase):
+    def setUp(self):
+      pass
+
+    def test_create_plain_bookable_name(self):
+        """Creating a plain bookable should create a simple bookable with default values and no extra features"""
+        b = Bookable.objects.create(name="Room A")
+        self.assertEqual(b.name, "Room A")
+        
+    def test_create_plain_bookable_description(self):
+        """Creating a plain bookable should create a simple bookable with default values and no extra features"""
+        b = Bookable.objects.create(name="Room A", description="A nice room")
+        self.assertEqual(b.description, "A nice room")
+        
+    def test_create_plain_bookable_public(self):
+        """Creating a plain bookable should create a simple bookable with default values and no extra features"""
```

Comment:
Could this be combined with `test_create_plain_bookable_name`? Otherwise the description could at least be updated.